submissions/jeff-anthony-tracey/scripts/article.js

```diff
@@ -116,14 +116,25 @@
         One for the author's name, and one for the total number of words across
         the matching articles written by the specified author. */
     return Article.allAuthors().map(function(a) { // 'a' is a reference to an individual author.
-      return {
-        // name:
-        // numWords: someCollection.filter(function(curArticle) {
+      var b = {
```

Comment:
Properly, you should switch this back to return {*object literal*} when you're done, the var b, console.log(b) stuff is just for debugging. return is clearer to read, which is important for things like this.